docs/dev/developer-guides/building-on-zksync/videos.md

```diff
@@ -0,0 +1,20 @@
+# Developer videos
+
+### zkSync 2.0 for Builders - Alex Gluchowski
+
+This talk covers everything you need to know about zkSync 2.0, and the user, and developer experience perspectives when building on zkSync.
+
+<iframe src="https://www.youtube.com/embed/xd2siik0iBU" title="zSync 2.0 for Builders - Alex Gluchowski" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### The zkEVM is Here - Steve Newcomb
+
+In this talk, Steve discusses how zkSync is going to be shipping the world's first zkEVM and how it's going to impact the blockchain space. 
+
+<iframe src="https://www.youtube.com/embed/QkZUlqetTRA" title="The zkEVM is Here - Steve Newcomb" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Antonio made some comments on autoplay and video size.... please review them all